web/src/stream_popover.ts

```diff
@@ -1152,6 +1181,15 @@ export async function build_move_topic_to_stream_popover(
                 update_move_messages_count_text(selected_option, message?.id);
             });
         }
+        if (stream_data.can_use_empty_topic_only(current_stream_id)) {
+            const $topic_input = $<HTMLInputElement>(
+                "#move_topic_form input.move_messages_edit_topic",
+            );
+            $topic_input.val("");
+            $topic_input.prop("disabled", true);
+            $topic_input.addClass("empty-topic-only");
```

Comment:
Can this code to disable the topic input in move modal be extracted in a function?